lib/buildkite/builder/extensions/lib.rb

```diff
@@ -0,0 +1,15 @@
+module Buildkite
+  module Builder
+    module Extensions
+      class Lib < Extension
+        def prepare
+          lib_dir = Buildkite::Builder.root.join(Buildkite::Builder::BUILDKITE_DIRECTORY_NAME, 'lib')
+
+          if lib_dir.exist? && lib_dir.directory? && !$LOAD_PATH.include?(lib_dir)
```

Comment:
```suggestion
          if lib_dir.directory? && !$LOAD_PATH.include?(lib_dir)
```

Does `$LOAD_PATH.include?(lib_dir)` work when `lib_dir` is a `Pathname` and not a string? Do you need to coerce to string?

Reply:
It works. Rails also did the same and push pathname object to `$LOAD_PATH`.